scm/scm.go

```diff
@@ -62,6 +63,10 @@ type SCM interface {
 	RemoveMember(context.Context, *OrgMembershipOptions) error
 	// Lists all authorizations for authenticated user.
 	GetUserScopes(context.Context) *Authorization
+	// Lists all pending repository invites.
+	GetRepositoryInvites(context.Context, *RepositoryOptions) ([]*github.RepositoryInvitation, error)
```

Comment:
This should use a custom `scm`-specific return type to avoid leaking GitHub-specific types out of the `scm` package.